docs/general/ledger.md

```diff
@@ -15,11 +15,15 @@ browsers like Brave and Chrome.
 :::
 
 The Polkadot [Ledger][] application is compatible with both the Ledger Nano S and the Ledger Nano X
-devices. Ledger devices are hardware wallets that keep your secret secured on a physical device that
-does not expose it to your computer or the internet.
+devices. Ledger devices are hardware wallets that keep your secret key secured on a physical device that
+does not expose it to your computer or the internet. That is, even if you connect your nano via USB to your computer, the private keys will not be leacked. Ledger devices are also hierarchical deterministic wallets (HD wallets), that is:
 
+  - Deterministic means that there is only one seed phrase to generate all the accounts for different cryptocurrencies.
+  - Hierarchical means that the accounts are generated in a tree-like structure for different purposes.
+
+Ledger devices can be equipped with applications that are blockchain-specific. Such applications are usually develped by third parties and they enable the user to perform essential actions such as transacting and staking.
```

Comment:
As we are talking in general, we can keep the wording generic (staking is specific to certain blockchain networks)

```suggestion
Ledger devices can be equipped with applications that are blockchain-specific. Such applications are usually developed by third parties and they enable the user to transact securely on the blockchain network.
```